3rdparty/python/BUILD

```diff
@@ -0,0 +1,5 @@
+python_requirements(
```

Comment:
i'm a little grossed out by all this new stuff at the root directory that's more specifically about src/python
but I'm sure we can tweak it in future versions, absolutely not a blocker
🤷